GLOSSARY.md

```diff
@@ -2,48 +2,39 @@
 
 This glossary accompanies the [GitOps Principles](./PRINCIPLES.md), and other supporting documents in this repository.
 
-- ## Break Glass
-
-    The temporary suspension of GitOps principles, often accomplished by pausing automated [reconciliation](#reconciliation).
-    While these principles apply to typical operations, it may at times be necessary to temporarily pause reconciliation, for example during incident management activities.
-    In these cases, other modes of operations should be considered (e.g. manual intervention), followed by any necessary updates to the desired state declarations, and finally resuming reconciliation of the system with the updated declarations.
-    Pragmatic exceptions to these guiding principles are expected from time to time during the journey toward a system being fully managed by GitOps.
-
 - ## Continuous
 
-    By "continuous" we adopt the industry standard to mean that [reconciliation](#reconciliation) continues to happen, not that it must be instantaneous.
+    "Continuous" is intended to match the industry standard term: [reconciliation](#reconciliation) continues to happen, not that it must be instantaneous.
 
 - ## Declarative Description
 
-    Describing the desired state or behavior of a system without specifying how that state will be achieved, thereby separating configuration (the desired state) from the implementation (commands, API calls, scripts etc.) that actually achieves the desired state described in the declarative description.
+    A configuration that describes the desired operating state of a system without specifying procedures for how that state will be achieved. This separates configuration (the desired state) from the implementation (commands, API calls, scripts etc.) used to achieve that state.
 
 - ## Desired State
 
-    The aggregate of all configuration data for a system form its desired state which is defined as data sufficient to recreate the system so that instances of the system are behaviourally indistinguishable, but do not include the state of any data stored within the system, eg. database contents.
+    The aggregate of all configuration data needed to start, bootstrap, recreate, run, and recover a system. This does not include data stored within a system, eg. database contents.
 
 - ## Drift
 
```

Comment:
```suggestion
```
@scottrigby @todaywasawesome I agree with @squaremo re: removing Drift from `GLOSSARY.md`

Reply:
OK we addressed this in the other thread. PTAL at 988c758 and continue to give feedback if you think that doesn't address those concerns 🙏